TeamSnapSDK/SDK/CollectionJSON/TSDKCollectionObject.h

```diff
@@ -12,21 +12,23 @@
 #import "TSDKObjectsRequest.h"
 #import "TSDKCollectionQuery.h"
 #import "TSDKCollectionCommand.h"
-
+#import "TSDKPersistenceFilePath.h"
 @class TSDKCollectionJSON;
 
+
 @protocol TSDKCollectionObjectBundledDataProtocol <NSObject>
 
 + (NSURL *_Nullable)bundledFileURL;
 
 @end
 
-@interface TSDKCollectionObject : NSObject <NSCoding>
+@interface TSDKCollectionObject : NSObject <NSCoding, TSDKPersistenceFilePath>
 
 @property (nonatomic, strong) TSDKCollectionJSON *_Nullable collection;
```

Comment:
For property declarations, you can include `nullable` in the property descriptors:

`@property (nonatomic, strong, nullable) TSDKCollectionJSON *collection;`

Then just leverage `NS_ASSUME_NONNULL_BEGIN` for the rest. Just looks more obj-c.